addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -74,6 +74,8 @@
                 <we-button data-custom-field="datetime">Date &amp; Time</we-button>
                 <we-button data-custom-field="binary">File Upload</we-button>
             </we-select>
+            <we-input string="Height" data-step="1" data-select-attribute="" data-unit="row"
```

Comment:
If step = 1 => row
else => rows
Or rows by default.

Reply:
:smoking: Curious how felix will handle that...

Reply:
Challenge!

Reply:
Easy way out I put rows by default like days in popup